File: CFOnlineShop/CFOnlineShop/ViewController/CFShoppingCartController/CFShoppingCartHeaderView.m
//
//  CFShoppingCartHeaderView.m
//  CFOnlineShop
//
//  Created by chenfeng on 2018/7/26.
//  Copyright © 2018年 chenfeng. All rights reserved.
//

#import "CFShoppingCartHeaderView.h"

@implementation CFShoppingCartHeaderView

- (instancetype)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
        
        _label = [[UILabel alloc] initWithFrame:CGRectMake(0, 5, self.mj_w, 35)];
        _label.font = SYSTEMFONT(14);
        _label.textAlignment = NSTextAlignmentCenter;
        _label.textColor = KDarkTextColor;
        _label.text = @"推荐商品";
        [self addSubview:_label];
        
    }
    return self;
}

@end

File: SwiftOnlineShopDemo/SwiftOnlineShopDemo/ViewController/CFShoppingCartController/view/CFShoppingCartHeaderView.swift
//
//  CFShoppingCartHeaderView.swift
//  SwiftOnlineShopDemo
//
//  Created by chenfeng on 2019/4/29.
//  Copyright © 2019年 chenfeng. All rights reserved.
//

import UIKit

class CFShoppingCartHeaderView: UICollectionReusableView {
    
    var label: UILabel!
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        
        label = UILabel.init(frame: CGRect(x: 0, y: 5, width: self.mj_w, height: 35))
        label.font = UIFont.systemFont(ofSize: 14)
        label.textAlignment = .center
        label.textColor = UIColor.black
        label.text = "推荐商品";
        self.addSubview(label)
        
    }
    
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
    
}
